static/js/chart/draw_choropleth.ts

```diff
@@ -100,15 +100,23 @@ function getColorScale(
   const extent = d3.extent(Object.values(dataValues));
   const medianValue = d3.median(Object.values(dataValues));
   let domainValues: number[] = domain || [extent[0], medianValue, extent[1]];
-  if (statVar.indexOf("Temperature") >= 0) {
+  if (
+    statVar.indexOf("Temperature") >= 0 ||
```

Comment:
Nit: move this to the util function as a function like isTemperatureStatVar()?

Reply:
done